### PR TITLE
Change user from systemUser to core

### PR DESCRIPTION
**Pull request by eranco74**

### CODE REVIEW

**Comment by eranco74** [comment]
`
[root@hpe-dl580gen9-01 test-infra]# ssh -i ssh_key/key core@192.168.126.12
Fedora CoreOS 31.20200319.dev.1
Tracker: https://github.com/coreos/fedora-coreos-tracker
Discuss: https://discussion.fedoraproject.org/c/server/coreos/

`

File: internal/bminventory/inventory.go [hunk 1]
```diff
@@ -64,15 +64,7 @@ const ignitionConfigFormat = `{
 "ignition": { "version": "3.0.0" },
   "passwd": {
     "users": [
-      {
-        "groups": [
-          "sudo",
-          "docker"
-        ],
-        "name": "core",
-        "passwordHash": "$6$MWO4bibU8TIWG0XV$Hiuj40lWW7pHiwJmXA8MehuBhdxSswLgvGxEh8ByEzeX2D1dk87JILVUYS4JQOP45bxHRegAB9Fs/SWfszXa5."
```
Comment (romfreiman) [inline internal/bminventory/inventory.go:73]: What is it?

Comment (eranco74) [reply internal/bminventory/inventory.go:73]: qwerty

**Comment by eranco74** [comment]
In case the user didn't pass ssh public key I will not create the user.
I can create the user and allow login with password but unsure we want to.
@ronniel1 , @avishayt , @filanov , @romfreiman @oshercc , did we have the user password here on purpose?

**Comment by avishayt** [comment]
> In case the user didn't pass ssh public key I will not create the user.
> I can create the user and allow login with password but unsure we want to.
> @ronniel1 , @avishayt , @filanov , @romfreiman @oshercc , did we have the user password here on purpose?

No, we don't want password-based login

**Comment by romfreiman** [comment]
is it console password? if so, we might want for now - in care the vm will
not get network

**Comment by eranco74** [comment]
OK,
Currently the status is this:
We have password based-login both for core user.
In addition if the user supply  ssh public key we are adding ssh & password based login for systemUser user.
After this PR is merged we will have:
In case the user provided ssh public key we will have ssh and password based login.
As Rom said we might want the password based login for debuging network issues.
Note that this is only for the ISO in live CD, once the install start and the host reboot this configuration is deleted.

**Comment by avishayt** [comment]
The password-based login is via SSH, console, or both?  Is SSH configured to allow password auth?

**Comment by eranco74** [comment]
ssh is configured not to allow password based login.
The password will work only from console

**Comment by avishayt** [comment]
> ssh is configured not to allow password based login.
> The password will work only from console

OK excellent

**Pull request merged**